docs/book/src/reference/version-skew-policy.md

```diff
@@ -0,0 +1,60 @@
+# Cluster API Version and version skew support policy
+
+## Supported versions
+
+The Cluster API project maintains the Core Provider, the Kubeadm Bootstrap Provider, the Kubeadm Control Plane Provider. These run on a Management Cluster. The project aims to keep the current minor release compatible with the actively supported Kubernetes minor releases, i.e., the current release (N), N-1, and N-2. To find out the exact range of Kubernetes APIs the Management Cluster may run, see the tables below.
+
+### Core Provider - `cluster-api-controller`
+
+||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (v0.2)|Cluster API v1alpha3 (v0.3)|
```

Comment:
Do we want to include v1alpha1? IIRC there is no easy upgrade path other from v1alpha1 -> v1alpha2, I'd remove it